assets/scss/local.scss

```diff
@@ -1,3 +1,5 @@
+  @import './accessible-autocomplete.min.css'; 
```

Comment:
Actually, correct one is   **@import './accessible-autocomplete.min';**  otherwise it throws an error.
2024-07-05 09:52:51 [ESBuild] ✘ [ERROR] Can't find stylesheet to import.
2024-07-05 09:52:51 [ESBuild]   ╷
2024-07-05 09:52:51 [ESBuild] 1 │   @import './accessible-autocomplete'; 
2024-07-05 09:52:51 [ESBuild]   │           ^^^^^^^^^^^^^^^^^^^^^^^^^^^